package.json

```diff
@@ -50,12 +50,28 @@
           "label": "References: Search mode",
           "description": "Results come from text search and heuristics. To use a language server for precise results, click to go to the README and follow the deployment instructions."
         }
+      },
+      {
+        "id": "feedback",
+        "command": "feedback",
+        "title": "Submit basic code intel feedback",
```

Comment:
- Idea: Remove "basic" and make this apply to all code intel
- Use Google Forms pre-filled parameters to include info about whether they were using basic/precise, what language, etc.: https://trevorfox.com/2015/06/dynamically-pre-fill-google-forms-with-mailchimp-merge-tags/

Reply:
Done. This will show up on both precise and basic code intel, and will auto-fill these parameters: basic/precise, language, and file.